src/test/java/com/google/collegeplanner/CourseTest.java

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.package com.google.collegeplanner.data;
+
+package com.google.collegeplanner.data;
+
+import java.text.ParseException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.rules.ExpectedException;
+
+/** Tests Course */
+@RunWith(JUnit4.class)
+public final class CourseTest {
+  private String[] gradingSystem;
+  private Course course;
+  @Rule
+  public ExpectedException exceptionRule = ExpectedException.none();
```

Comment:
does this need to be "reset" before every test?

Reply:
I moved the initialization into the before method just in case